Feat/add pymechanical

rs-bh-n opened the pull request:
added pymechanical cheat sheet

Revathyvenugopal162 commented:
Thank you @rs-bh-n for opening the PR, please provide the screenshot/pdf file of the PDF generated  in the description of the PR, so that it will be easy for docs team for review. Adding @EstherProsperi for the proof reading.

rs-bh-n commented:
Hi @Revathyvenugopal162 and @EstherProsperi  , here is the snapshot of the pdf generated.

![image](https://github.com/ansys/pyansys-cheat-sheet/assets/96760708/44be3f88-95f3-4d70-af83-527e0b4d50ab)

EstherProsperi commented:
> Hi @Revathyvenugopal162 and @EstherProsperi , here is the snapshot of the pdf generated.
> 
> ![image](https://user-images.githubusercontent.com/96760708/254480814-44be3f88-95f3-4d70-af83-527e0b4d50ab.png)

Any chance I could have the actual PDF?

rs-bh-n commented:
[pymechanical_cheat_sheet.pdf](https://github.com/ansys/pyansys-cheat-sheet/files/12092267/pymechanical_cheat_sheet.pdf)

Here it is , @EstherProsperi .

EstherProsperi commented:
> [pymechanical_cheat_sheet.pdf](https://github.com/ansys/pyansys-cheat-sheet/files/12092267/pymechanical_cheat_sheet.pdf)
> 
> Here it is , @EstherProsperi .

[pymechanical_cheat_sheet.pdf](https://github.com/ansys/pyansys-cheat-sheet/files/12093755/pymechanical_cheat_sheet.pdf)

Revathyvenugopal162 commented:
@rs-bh-n can you please adapt the review from Ester? if you encounter any issue/ roadblock, let me know i will try to adapt Thanks.

Revathyvenugopal162 commented on makefile:
```diff
@@ -1,7 +1,6 @@
 
-CHEATSHEETS   = pymapdl_cheat_sheet pyfluent_cheat_sheet pyaedt_API_cheat_sheet pyedb_API_cheat_sheet pyprimemesh_cheat_sheet pydpf-core_cheat_sheet pydynamicreporting_cheat_sheet
-
+CHEATSHEETS   = pymapdl_cheat_sheet pyfluent_cheat_sheet pyaedt_API_cheat_sheet pyedb_API_cheat_sheet pyprimemesh_cheat_sheet pydpf-core_cheat_sheet pymechanical_cheat_sheet pydynamicreporting_cheat_sheet
```
```suggestion
CHEATSHEETS=pymapdl_cheat_sheet pyfluent_cheat_sheet pyaedt_API_cheat_sheet pyedb_API_cheat_sheet pyprimemesh_cheat_sheet pydpf-core_cheat_sheet pymechanical_cheat_sheet pydynamicreporting_cheat_sheet
```

rs-bh-n commented:
Thanks for your guidance , @Revathyvenugopal162  , in git push.

@EstherProsperi , Thanks for your inputs on wednesday :  Here is the pdf with the below changes . Let me know if this is okay.

[pymechanical_cheat_sheet.pdf](https://github.com/ansys/pyansys-cheat-sheet/files/12122538/pymechanical_cheat_sheet.pdf)


1) Titles changed(same as in docs now. with underscores intact) : 
A.Connect to a Remote Session of Mechanical from Python
B.Load an Embedded instance of Mechanical in Python

2) Retained the title "Launch and Connect to a Session " ( a and b are methods of doing this )

3) Local and Remote : Reworded , comment modified

4) Subtext changes :
	"Example: Import file and print the count of bodies"
	"Example: Import file and print the count of bodies"
	"To turn on logging for warnings:"
	
5)python corrections : added missing import command .added black format .and  set the reference links as two rows.

rs-bh-n commented:
Thanks @EstherProsperi , for your inputs.
Attached are the pdfs.

@Revathyvenugopal162 , the documentation approval is done.

Revathyvenugopal162 reviewed (approved):
Looks good . Thankyou for the cheatsheet @rs-bh-n and thank you @EstherProsperi for the quick and thorough review,

Revathyvenugopal162 commented:
merging!
